code/controllers/subsystems/virtual_reality.dm

```diff
@@ -173,7 +188,7 @@ SUBSYSTEM_DEF(virtualreality)
 	if(null_vr_mob)
 		target.vr_mob = null
 
-/datum/controller/subsystem/virtualreality/proc/mech_selection(var/user, var/network)
+/datum/controller/subsystem/virtualreality/proc/mech_selection(var/user, var/network, var/return_list)
```

Comment:
`return_list` seems useless here, you can always return the list as it doesn't seem you return anything else